docs/kr.tree

```diff
@@ -278,6 +278,7 @@
             <toc-element id="components-stability.md"/>
             <toc-element id="components-stability-pre-1.4.md"/>
             <toc-element toc-title="Compatibility">
+                <toc-element id="compatibility-guide-15.md"/>
```

Comment:
```suggestion
```

Let's hide the Guide in TOC until the release. Otherwise, this can confuse people: they'll think that 1.5.0 is already released.